src/builtin/tables.scrbl

```diff
@@ -47,53 +67,150 @@ my-table = table: name :: String, age :: Number, favorite-color :: String
 end
   }
 
-Note that @pyret{my-table} is referred to in many of the following examples.
+Entering the name of a table in the interactions window after running the program
+or using the @pyret{print} function will display a formatted version of the table.
+
+@(image "src/builtin/table-print.png")
+  
+@margin-note{Note that @pyret{my-table} is referred to in many of the
+following examples.}
 
   @section[#:tag "s:tables:loading"]{Loading Tables}
 
-Table loading expressions allow for the importing of tables from the
-outside world into Pyret. Currently, only Google Sheets is supported.
-@;(see the @tt{gdrive-sheets} module documentation for details)
-In addition to data sources, the notion of @italic{sanitizers} is used. These
-are used to properly load each entry of the table as the correct type;
-for example, the @pyret{string-sanitizer} in the @tt{data-source} module
-causes each item in its column to be loaded as a string (if it is not a
-string, it is first converted to one). This is illustrated by the following
-example:
+Almost any practical use of a @pyret{Table} will involve loading
+data from an external spreadsheet.  Currently, only importing
+from Google Sheets is supported.
+
+@margin-note{Data sources are currently an internal concept to Pyret, so no public
+interface for creating them is yet supported.
+
+You can import most relevant file types, including .xlsx,
+into Google Sheets, and then into Pyret, so you should be able to
+get almost any tabular data into Pyret with a little effort.}
+
+As a simple and consistent example, let's say we wanted to import the
+@tt{my-table} data from a spreadsheet:
+
+@(image "src/builtin/gsheet-1.png")
+
+@margin-note{Pyret tables and sanitizers don't understand or recognize
```

Comment:
Only headers.  When using gsheets as data sources, the assumption is that they're neatly well-formatted, rather than human-generated with crappy partial columns and comments and whatnot.